framework/doc/content/source/base/Assembly.md

```diff
@@ -1,3 +1,42 @@
-!! MOOSE Documentation Stub: Remove this line when content is added.
-
 # Assembly
+
+The `Assembly` class holds most routines and objects related to assembling the numerical
+system, such as the ones listed below. In the non-exhaustive list below, we refer to as
+`local` the current element, face, quadrature points and neighbor values of each quantity.
+
+- the current element
+
+- the neighbor element
+
+- the local volumes
+
+- the Jacobian weights
+
+- the local quadrature rules and points
+
+- the local values of current variable, its spatial derivatives at first and second order,
+  its curl on the element
```

Comment:
Do you want these new lines between every list item? Does that even format correctly?

Reply:
formats fine https://mooseframework.inl.gov/docs/PRs/18866/site/source/base/Assembly.html
I ve found that more new lines work better than less with those lists tbh

Reply:
better how?

Reply:
if you delete the new line before or one of the new lines after it wont work. It ll work fine within maybe

Reply:
what would you add to Assembly?

Reply:
Assembly is many thousands of lines of code haha. But I would hope that our users don't often have to touch assembly methods, if ever. So I'm not sure whether we need a lot of user documentation for that object. Doxygen and code comments are more important IMO. In that vein, I think the intro material you have here is good

Reply:
A one or zero new lines between list items should not render any differently. If you put two new lines you get a new list.

Reply:
I m happy with a single new line. Unless you guys feel differently I was not planning on changing this

Reply:
I feel very strongly. Jk